tests/docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: '2'
+services:
+  codecept:
+    image: testing/codeception/codeception
+    build: ..
+    volumes:
+      - ../tests:/repo/tests
+      - ../codeception.yml:/repo/codeception.yml
+      - ../src:/repo/src
+      - ../shim.php:/repo/shim.php
+    ports:
+      - '80'
+  test1:
+    extends:
+      service: codecept
+
+  firefox:
+    image: selenium/standalone-firefox-debug:2.52.0
+  chrome:
+    image: selenium/standalone-chrome-debug:2.52.0
+  redis:
```

Comment:
please also add mongo to it and instructions to tests/readme.md on how to run tests through docker compose